app/templates/mock-login.hbs

```diff
@@ -1,54 +1,37 @@
-<div id="main" itemprop="mainContentOfPage" role="main" tabindex="-1" class="au-c-main-container__content au-c-main-container__content--scroll">
+<AuBodyContainer @scroll={{true}} itemprop="mainContentOfPage">
+  <main id="content">
   <AuContentHeader @titlePartOne="Vlaanderen" @titlePartTwo="is lokaal bestuur">
     <img sizes="50vw" src="/assets/images/inzendingen-header-1600.jpg" srcset="/assets/images/inzendingen-header-320.jpg 320w, /assets/images/inzendingen-header-1024.jpg 1024w, /assets/images/inzendingen-header-1600.jpg 1600w" alt="" role="presentation">
   </AuContentHeader>
 
-  <section class="au-o-region-large">
-    <div class="au-o-layout au-o-layout--small">
+  <div class="au-o-region-large">
+    <div class="au-o-layout">
 
-      <AuHeading @level="2" @skin="3">Kies een bestuurseenheid om mee in te loggen.</AuHeading>
+      <AuHeading @skin="4">Kies een bestuurseenheid om mee in te loggen.</AuHeading>
+      <p class="au-u-margin-top-small au-u-margin-bottom">
+        {{#let (unique-id) as |id|}}
+        <AuLabel for={{id}}>Bestuurseenheid zoeken</AuLabel>
+        <input id={{id}} class="au-c-input au-c-input--block js-js-input-pattern-bound" placeholder="Aalst, Berchem, &hellip;" value={{this.gemeente}} oninput={{perform this.updateSearch value='target.value'}}>
+        {{/let}}
+      </p>
 
-      <div class="au-o-region">
         <MockLogin as |login|>
           {{#if this.queryStore.isRunning}}
-            <AuLoader @size="small" />
-            <span class="au-u-hidden-visually">Aan het laden</span>
+            <AuLoader @padding="small" />
           {{else}}
-            {{#if login.errorMessage}}
-              <AuAlert @icon="info-circle" @title={{login.errorMessage}}  @skin="error">{{login.errorMessage}}</AuAlert>
+            {{#if this.login.errorMessage}}
+              <AuAlert @alertIcon="alert-triangle" @alertTitle={{this.login.errorMessage}} @alertSkin="warning" />
```

Comment:
I'm pretty sure this triggers deprecations at the moment. The `alertX` arguments are deprecated.